### ISSUE TITLE
Clean up tmp files

### ISSUE THREAD

**Issue by alanhussey** [alanhussey/golden-hour#10]
Make sure to delete the timelapse photos and the completed video after a successful upload.

**Comment by jason-curtis** [comment]
Great point! It would be nice to be able to turn this off, too, though. Rather than having the raw data disappear, I might want to back them up elsewhere for other projects :thinking:

**Comment by alanhussey** [comment]
Good point! I think it's also valuable to leave them around for debugging. I was imagining a monthly(?) script that would clear out photos and videos more than a month old.
